actions/views/channel.test.js

```diff
@@ -519,4 +519,15 @@ describe('channel view actions', () => {
             expect(markChannelAsRead).not.toHaveBeenCalled();
         });
     });
+
+    describe('updateToastStatus', () => {
+        test('should disptach updateToastStatus action with the true as argument', async () => {
+            await store.dispatch(Actions.updateToastStatus(true));
+
+            expect(store.getActions()).toEqual([{
+                data: true,
+                type: 'UPDATE_TOAST_STATUS'
```

Comment:
slightly concerned about the naming here: what should someone think of "update_toast_status=true"? maybe we should rename into something more descriptive?

Reply:
i am 0/5 on this but this is not something that people stumble up on when reading code IMO.